### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by GalackQSM** [WongKit/ft-launcher#1]
Could not get remote checksum file: https://pokemonworldlegends.fr/launcher/files.md5

**Comment by WongKit** [comment]
The file https://pokemonworldlegends.fr/launcher/files.md5 does not exist. Have you forgotten to create or upload it? You can do so by holding the shift key while starting the launcher.

**Comment by GalackQSM** [comment]
I have already done them is put the folders and files in the update folders of my FTP my nothing works

**Comment by WongKit** [comment]
This is strange. Could you upload a sample files.md5 again so that I can check? The error message "Could not get checksum..." also appears, if the application is blocked by a firewall for example.
